lib/utils/yaml-utils.ts

```diff
@@ -12,7 +13,8 @@ export function readYamlFromDir(dir: string, cluster: eks.Cluster): void {
                         if (data != undefined) {
                             let i = 0;
                             yaml.loadAll(data).forEach((item) => {
-                                cluster.addManifest(file.substr(0, file.length - 5) + i, item);
+                                const resources = cluster.addManifest(file.substr(0, file.length - 5) + i, item);
```

Comment:
Since you are not making changes to ensure the yamls go into the namespace, they may end up in default. We need an explicit statement in the docs (and code docs) that customer is responsible for namespace settings in the yamls.

Reply:
This has been corrected under the new PR: #185. Specifically, comments have been added to code base, and the team.md documentation also alerts users to check for the correct namespaces in the manifest files.